pdp/__init__.py

```diff
@@ -15,6 +15,7 @@
 import yaml
 import static
 from beaker.middleware import SessionMiddleware
+from git import Repo
```

Comment:
So, I initially liked this approach. However, the big, big drawback is that with this change, you would *always* need to run the code out of an existing git repository. This would basically eliminate the ability to run the code as a [source distribution](https://packaging.python.org/distributing/#source-distributions) or a built [wheel](https://packaging.python.org/distributing/#wheels).

I think that it would be a better choice to put all of the `get_commitish()` code in the setup.py to be run at build time or doc build time. That way we have the option of building our package on a dev machine, pushing the build to our PyPI repo, and installing from there on a prod machine (that wouldn't necessarily have git installed).